src/utils/__tests__/narrow-test.js

```diff
@@ -322,7 +329,8 @@ describe('getNarrowFromMessage', () => {
     expect(actualNarrow).toEqual(expectedNarrow);
   });
 
-  test('if recipient of a message is string, returns a stream narrow', () => {
+  test('for stream message with empty topic, returns a stream narrow', () => {
+    // TODO this behavior seems pretty dubious
```

Comment:
> `// TODO this behavior seems pretty dubious`

Indeed! 🙂 See https://github.com/zulip/zulip-mobile/commit/5bf51c7c14427508e00ea2fbf7dc4814da511302 on #M4317, which removes it.